frontend/src/components/upload/Upload.js

```diff
@@ -23,9 +23,12 @@ const Upload = (props) => {
           Both pieces of ID must show the same address. Image files must be in
           jpg or png format.
         </p>
-        <a href="/identificationExamples" target="_blank">
-          See examples of accepted ID
-        </a>
+        <p>
+          <a href="/identificationExamples" target="_blank">
+          See examples of accepted ID 
+          </a>
+         (Link opens in a new tab to prevent losing form data)
```

Comment:
Link needs to be lowercase, (link opens in a new tab to prevent losing form data)